src/window.rs

```diff
@@ -1367,14 +1367,31 @@ impl Window {
         self.window.maybe_queue_on_main(move |w| w.set_theme(theme))
     }
 
-    /// Returns the current window theme override.
+    /// Returns the current system theme.
     ///
-    /// Returns `None` if the current theme is set as the system default, or if it cannot be
-    /// determined on the current platform.
+    /// Returns `None` if it cannot be determined on the current platform.
+    ///
+    /// This is the same as [`ActiveEventLoop::system_theme()`], and is provided for convenience.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Wayland / x11 / Orbital:** Unsupported.
+    ///
+    /// [`ActiveEventLoop::system_theme()`]: crate::event_loop::ActiveEventLoop::system_theme
+    #[inline]
+    pub fn system_theme(&self) -> Option<Theme> {
```

Comment:
Do we really need it on the window? In general, if the system theme is tied to event loop, I'd keep it there, it's not like you have to poll it very often etc, and it would require changes internally to get it on the window, well, at least, sometimes.

Reply:
I can definitely imagine it being useful. Right now it comes with no downsides, but yes, if it requires further changes internally then lets just remove it when it comes to that.

Would advocate to remove it if it makes problems in an implementation or when we make `Window` not `Send + Sync` anymore, but keep it for now.

Reply:
> We can always add it if there's a desire for it, but I'd not add _more_ without a request.

That's pretty convincing to me.

Reply:
Done.